frontend/src/component/menu/routes.ts

```diff
@@ -337,6 +338,14 @@ export const routes: IRoute[] = [
         type: 'protected',
         menu: {},
     },
+    {
+        path: '/integrations/view/:providerId',
```

Comment:
Currently this serves only static page on Jira, but if more pages come, there might be possibility to make it generic.